docs/contributor/howto/index.md

```diff
@@ -1,3 +1,9 @@
+---
+myst:
+  html_meta:
+    description: "Explore step-by-step how-to guides for Juju developers covering testing, debugging, MongoDB management, and cross-architecture development."
```

Comment:
I think we should include Dqlite in here as that is also a DB that we use and have docs for in 3 branches.

Reply:
4.0 forward should drop Mongo related wording.

Reply:
Rephrased as "database". That captures the intent and should preempt the need for adjustments in the forward merge.